README.md

```diff
@@ -96,7 +96,7 @@ Thanks to [search-query-parser](https://github.com/nepsilon/search-query-parser)
 
 The MIT License (MIT)
 
-Copyright (c) 2017 Mixmax <marcus@mixmax.com>
+Copyright (c) 2017 Mixmax
```

Comment:
I think we do this in other repos, not sure.
```suggestion
Copyright (c) 2017 Mixmax, Inc
```

Reply:
ah yes, added